idb/postgres/internal/schema/setup_postgres.sql

```diff
@@ -117,3 +117,11 @@ CREATE TABLE IF NOT EXISTS account_app (
 
 -- For looking up existing app local states by account
 CREATE INDEX IF NOT EXISTS account_app_by_addr_partial ON account_app(addr) WHERE NOT deleted;
+
+-- For looking up app box storage
+CREATE TABLE IF NOT EXISTS app_box (
+  app bigint NOT NULL,
+  name bytea NOT NULL,
```

Comment:
Are these names expected to be human readable?

Reply:
No. They can be any bytestring whatsoever. We're not sure what the usage pattern will be, but I personally expect many of the names to be human readable, if not the values.